python/logview.py

```diff
@@ -218,6 +163,9 @@ def main():
             e = InteractiveShellEmbed(user_ns = user_ns, user_module = user_module)
             e.enable_pylab(import_all = True)
             e("Analyzing log file: %s" % src)
+
 #-------------------------------------------------------------------------------
 if __name__ == "__main__":
+    #import cProfile
```

Comment:
Can I pretty please keep this for awhile, while I'm still performance tuning and want to uncomment it occasionally?